### DISCUSSION TITLE
Github actions selfhosted runners on enterprise server

### DISCUSSION THREAD

**Discussion by geekette86** [community/community#46848] [Enterprise]
### Select Topic Area

Question

### Body

Hello i am trying to create a Token with this url `curl -vvv POST https://hostname/api/v3/enterprises/myentreprise/actions/runners/registration-token --header "authorization: Bearer <token>" --header 'content-type: application/json'`
the token has `manage_runners:enterprise` scope but i always have this response from the API
```
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/enterprise/3.5/rest/reference/actions#get-a-self-hosted-runner-for-an-enterprise"
}
Can you please check this because based on your documentation everything i did was ok 
regards
```

**Comment by Hosain559** [comment]
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/enterprise/3.5/rest/reference/actions#get-a-self-hosted-runner-for-an-enterprise"
}
Can you please check this because based on your documentation everything i did was ok 
regards

**Comment by Hosain559** [comment]
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/enterprise/3.5/rest/reference/actions#get-a-self-hosted-runner-for-an-enterprise"
}
Can you please check this because based on your documentation everything i did was ok 
regards
